Tests/FDBTests/FDBTests.swift

```diff
@@ -123,6 +130,58 @@ class FDBTests: XCTestCase {
         let result = try fdb.get(key: updatedKey)
         XCTAssertNotNil(result)
         try XCTAssertEqual(result!.cast() as String, value)
+        
+        
+        let asyncKey = subspace[FDB.Versionstamp(userData: 42)]["myAsyncKey"]
+        let asyncValue: String = "world!"
+        
+        let versionstampedKeyValueFuture: EventLoopFuture<String?> = (fdb.withTransaction(on: self.eventLoop) { transaction in
+            return transaction
+                .set(versionstampedKey: asyncKey, value: getBytes(asyncValue.utf8))
+                .flatMap { transaction in
+                    return transaction.getVersionstamp(commit: true)
```

Comment:
You don't really need this `transaction` argument at 141, as it's the same instance as on line 138. `set` methods are designed to return future to transaction for cases when user wants to handle the transaction manually, without `withTransaction`.